### PR TITLE
Leaderboard by songs guessed and games played

### PR DESCRIPTION
**Pull request by taahamahdi**

### CODE REVIEW

File: src/commands/game_commands/leaderboard.ts [hunk 1]
```diff
@@ -28,6 +28,12 @@ const logger = new IPCLogger("leaderboard");
 export const ENTRIES_PER_PAGE = 10;
 
 export enum LeaderboardType {
+    EXP = "exp",
+    GAMES_PLAYED = "games_played",
```
Comment (Brainicism) [inline src/commands/game_commands/leaderboard.ts:32]: Should these be hyphens?

File: src/commands/game_commands/leaderboard.ts [hunk 2]
```diff
@@ -169,14 +182,50 @@ export default class LeaderboardCommand implements BaseCommand {
         if (parsedMessage.components.length === 1) {
             LeaderboardCommand.showLeaderboard(
                 message,
-                type ?? LeaderboardType.GLOBAL,
+                type ?? LeaderboardType.EXP,
```
Comment (Brainicism) [inline src/commands/game_commands/leaderboard.ts:185]: Add these as default values to the function instead

**Review by Brainicism — CHANGES_REQUESTED** [review]

File: src/commands/game_commands/leaderboard.ts [hunk 3]
```diff
@@ -28,13 +28,19 @@ const logger = new IPCLogger("leaderboard");
 export const ENTRIES_PER_PAGE = 10;
 
 export enum LeaderboardType {
+    EXP = "exp",
+    GAMES_PLAYED = "gamesplayed",
```
Comment (Brainicism) [inline src/commands/game_commands/leaderboard.ts:32]: Can this just be 'songs' and 'games'?

Comment (taahamahdi) [reply src/commands/game_commands/leaderboard.ts:32]: I initially considered it, but decided against it because `game` is already an argument

**Review by Brainicism — APPROVED** [review]

File: src/commands/game_commands/leaderboard.ts (hunk 3 above)
Comment (taahamahdi) [reply src/commands/game_commands/leaderboard.ts:32]: We can alternatively separate this into three commands, and maintain the logic in this file

**Pull request merged**